tests/testthat/setup.R

```diff
@@ -21,6 +21,16 @@ compiled_site_inf_model <- cmdstanr::cmdstan_model(
   dir = tempdir()
 )
 
+spatial_fxns <- cmdstanr::cmdstan_model(
+  stan_file = system.file(
+    "stan", "functions", "spatial_functions.stan",
+    package = "wwinference"
+  ),
+  compile = TRUE,
```

Comment:
Suggest adding the `dir = tempdir()`  as in the compilation of the `wwinference.stan` model, but otherwise this looks right!